src/Page/Index.elm

```diff
@@ -98,5 +98,61 @@ view _ _ _ =
             , Html.p []
                 [ Html.text "Gren produces small JavaScript files, and runs surprisingly fast. Applications written in Gren can be both smaller and faster than your average React-based application." ]
             ]
+        , Html.article
```

Comment:
Move this above `Compiler as an assistant` and `Efficiency`? We want to show people code as early as possible.